### PR TITLE
Marking and unmarking important skills command

### LINKED ISSUE #72: As a student who formed project groups and prioritizing certain skill sets, I can mark some skills as more important than others
...so that I know which skills I need to prioritize

### LINKED ISSUE #73: [Logic] Add marking functionality to skills in groups
Implement the following comands:

- mark-important GROUP_NAME [-s SKILL]...
- unmark-important GROUP_NAME [-s SKILL]...

### PR DESCRIPTION
**Pull request by Ryan-Reno**
Added marking and unmarking important skills functionality
[!] at the front signifies that it is an important skill

Closes #72 
Closes #73

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -118,6 +118,31 @@ public static Set<Skill> parseSkills(Collection<String> skills) throws ParseExce
...
     }
 
+
+    /**
+     * Parses a {@code String skill} into a {@code Skill} with importance.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code skill} is invalid.
+     */
+    public static Skill parseSkillImportant(String skill) throws ParseException {
```
Comment (ivan24004) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:128]: Would it be better to just use the regular `parseSkill` and set the `important` flag to `true`?

**Review by ivan24004 — APPROVED** [review]
Minor comments regarding some design decisions. Otherwise, ready for merging.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (Ryan-Reno) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:128]: Yeah did that for Unmarking but forgot to change it for Marking. Will change it once i get back

File: src/test/data/JsonSerializableContactListTest/duplicateCourseMateContactList.json [hunk 2]
```diff
@@ -3,8 +3,12 @@
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "alice@example.com",
-    "telegramHandle": "alice",
-    "skills": [ "Java" ]
+    "address": "123, Jurong West Ave 6, #08-111",
```
Comment (yuhinarita) [inline src/test/data/JsonSerializableContactListTest/duplicateCourseMateContactList.json:6]: Wait why is there an address field?

**Pull request merged**